easybuild/easyconfigs/c/CUDA/CUDA-9.0.176.eb

```diff
@@ -0,0 +1,18 @@
+name = 'CUDA'
+version = '9.0.176'
+
+homepage = 'https://developer.nvidia.com/cuda-toolkit'
+description = """CUDA (formerly Compute Unified Device Architecture) is a parallel
+ computing platform and programming model created by NVIDIA and implemented by the
+ graphics processing units (GPUs) that they produce. CUDA gives developers access
+ to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://developer.nvidia.com/compute/cuda/%(version_major_minor)s/Prod/local_installers/']
+sources = ['cuda_%(version)s_384.81_linux-run']
+checksums = ['96863423feaa50b5c1c5e1b9ec537ef7ba77576a3986652351ae43e66bcd080c']
+
+modextravars = {'CUDA_HOME': '%(installdir)s'}
```

Comment:
@smoors This is no longer needed, it's handled by the `EB_CUDA` easyblock since EasyBuild 2.5.0

Reply:
@boegel you mean I can remove the last line, right?

Reply:
after adding the EB_CUDA easyblock

Reply:
No need to specify it to use `EB_CUDA`, it already does (if no `easyblock =` is specified, EasyBuild will use the software-specific easyblock based on `name`).

Just drop the `modextravars` line (and the empty line below it).